src/functionality.js

```diff
@@ -0,0 +1,35 @@
+export const setLocalStore = (todoTasks) => {
+  localStorage.setItem('tasks', JSON.stringify(todoTasks));
+};
+
+export const addTask = (desctiption, completed = false, index, todoList) => {
+  const task = {
+    desctiption,
+    completed,
+    index,
+  };
+  todoList.todoTasks.push(task);
+  // displayTasks();
+  setLocalStore(todoList.todoTasks);
+};
+
+export const removeTask = (i, todoList) => {
+  todoList.todoTasks.splice(i, 1);
+  todoList.todoTasks.forEach((task) => {
+    task.index = i + 1;
+  });
+  // displayTasks();
```

Comment:
- [ ] please remove all the commented out code in order to keep your codebase clean and readable.